payjoin/src/send/mod.rs

```diff
@@ -321,7 +321,7 @@ impl RequestContext {
     #[cfg(feature = "v2")]
     pub fn extract_v2(
         &mut self,
-        ohttp_proxy_url: &str,
+        ohttp_proxy: Url,
```

Comment:
@jbesraa I'm pausing merging this to leave time for you to consider this since it affects the API